snapcraft/plugins/crystal.py

```diff
@@ -80,7 +94,12 @@ def build(self):
         super().build()
 
         self.run(["shards", "install", "--production"], self.builddir)
-        self.run(["shards", "build", "--production"], self.builddir)
+
+        build_flags = ["--production"]
```

Comment:
we should probably let the defaults take place here, how would I build a non production build if I wanted to?

Reply:
Do you mean here?

```
schema["properties"]["crystal-buildflags"] = {
            "type": "array",
            "minitems": 1,
            "uniqueItems": True,
            "items": {"type": "string"},
            "default": [], # << HERE
        }
```

Reply:
That is a good place, please make mention of the default in the docstring.

Reply:
Done

Reply:
You seem to have missed adding it to "default" in the schema

Reply:
Done

Reply:
ok, so now this line is not needed as it should be taken into account in the conditional right below